decidim-forms/app/types/decidim/forms/questionnaire_type.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Forms
+    QuestionnaireType = GraphQL::ObjectType.define do
+      name "Questionnaire"
```

Comment:
Maybe it would be good to add to the changelog that this PR also adds API specifications for decidim-form questionnaires..

Reply:
It does not, that was because this PR was dependant on meetings